assets/airports/ltba.json

```diff
@@ -1,254 +1,6866 @@
+            "exitPoints": {
+                "ERTAS": []
+            },
+            "draw": [
+                ["_RWY35R", "_IST177005", "CEK", "ERTAS"],
+                ["_RWY35L", "_IST177005", "CEK", "ERTAS"]
+            ]
+        },
+        "ERTAS1T": {
+            "icao": "ERTAS1T",
+            "name": "Ertas One Tango",
+            "rwy": {
+                "LTBA17L": [],
+                "LTBA17R": []
+            },
+            "body": [["^_IST177005", "A14+"], "BKZ", "ERTAS"],
+            "exitPoints": {
+                "ERTAS": []
+            },
+            "draw": [
+                ["_RWY35R", "_IST177005", "BKZ", "ERTAS"],
+                ["_RWY35L", "_IST177005", "BKZ", "ERTAS"]
+            ]
+        },
+        "ERTAS1U": {
+            "icao": "ERTAS1U",
+            "name": "Ertas One Uniform",
+            "rwy": {
+                "LTBA35L": []
+            },
+            "body": [["^_IST356007", "A16+"], "_ERTAS204018.5", "ERTAS"],
+            "exitPoints": {
+                "ERTAS": []
+            },
+            "draw": [
+                ["_RWY17R", "_IST356007", "_ERTAS204018.5", "ERTAS"]
+            ]
+        },
+        "FENER1D": {
+            "icao": "FENER1D",
+            "name": "Fener One Delta",
+            "rwy": {
+                "LTBA35L": []
+            },
+            "body": [["^_IST356007", "A16+"], "_BKZ283015.5", "FENER"],
+            "exitPoints": {
+                "FENER": []
+            },
+            "draw": [
+                ["_RWY17R", "_IST356007", "_BKZ283015.5", "FENER"]
+            ]
+        },
+        "FENER1K": {
+            "icao": "FENER1K",
+            "name": "Fener One Kilo",
+            "rwy": {
+                "LTBA35R": []
+            },
+            "body": [["^_IST356007", "A16+"], "_BKZ283015.5", "FENER"],
+            "exitPoints": {
+                "FENER": []
+            },
+            "draw": [
+                ["_RWY17L", "_IST356007", "_BKZ283015.5", "FENER"]
+            ]
+        },
+        "FENER1L": {
+            "icao": "FENER1L",
+            "name": "Fener One Lima",
+            "rwy": {
+                "LTBA35R": []
+            },
+            "body": [["^_IST356007", "A16+"], "_IST309014.5", "FENER"],
+            "exitPoints": {
+                "FENER": []
+            },
+            "draw": [
+                ["_RWY17L", "_IST356007", "_IST309014.5", "FENER"]
+            ]
+        },
+        "FENER1N": {
+            "icao": "FENER1N",
+            "name": "Fener One November",
+            "rwy": {
+                "LTBA05": []
+            },
+            "body": [["^_IST051004.5", "A10+"], "_IST309006", "FENER"],
+            "exitPoints": {
+                "FENER": []
+            },
+            "draw": [
+                ["_RWY23", "_IST051004.5", "_IST309006", "FENER"]
+            ]
+        },
+        "FENER1U": {
+            "icao": "FENER1U",
+            "name": "Fener One Uniform",
+          
```

Comment:
Line 2853:
No need for square brackets around _IST245005
No need for fly-over along the arc

Reply:
Why no need for square brackets? There's a restriction 1400 or above at IST245005
I tried to realize the arc without fly-over fixes, didn't work the aircraft just skipped the fixes.
I tested out multiple things and that was the best result I was able to get.